examples/token/nep17/nep17.go

```diff
@@ -64,6 +66,9 @@ func (t Token) Transfer(ctx storage.Context, from, to interop.Hash160, amount in
 	totalAmountTo := amountTo + amount
 	storage.Put(ctx, to, totalAmountTo)
 	runtime.Notify("Transfer", from, to, amount)
+	if to != nil && management.GetContract(to) != nil {
+		contract.Call(to, "onNEP17Payment", contract.All, from, amount, data)
```

Comment:
Don't we need to check that onPayment exists? Otherwise it will cause transfer failure.

Reply:
And that's exactly what should happen if we're transferring tokens to contract that isn't ready to accept them.